specs/stork-deployment.yaml

```diff
@@ -76,8 +76,8 @@ rules:
     resources: ["rules"]
     verbs: ["get", "list"]
   - apiGroups: ["stork.libopenstorage.org"]
-    resources: ["clusterpairs", "migrations", "groupvolumesnapshots", "storageclusters"]
-    verbs: ["get", "list", "watch", "update", "patch"]
+    resources: ["clusterpairs", "migrations", "groupvolumesnapshots", "storageclusters", "schedulepolicies", "migrationschedules"]
```

Comment:
While writing the integration test, I also run into more missing permissions in the stork-role. configmaps, services, secrets, namespaces will need list permissions. 
```
  Warning  Failed      2m                stork  Error migrating resources: configmaps is forbidden: User "system:serviceaccount:kube-system:stork-account" cannot list configmaps in the namespace "mysql-1-pvc-mysql-migration"
  Warning  Failed      1m                stork  Error migrating resources: secrets is forbidden: User "system:serviceaccount:kube-system:stork-account" cannot list secrets in the namespace "mysql-1-pvc-mysql-migration"
  Warning  Failed      13s (x3 over 1m)  stork  Error migrating resources: services is forbidden: User "system:serviceaccount:kube-system:stork-account" cannot list services in the namespace "mysql-1-pvc-mysql-migration"
```

Reply:
Actually I was missing the below in my stork-role. This is already present on install.portworx.com.

```
  - apiGroups: ["*"]
    resources: ["*"]
    verbs: ["list", "get"]
```